test/utils.js

```diff
@@ -167,8 +167,9 @@ module.exports.filesAfter = function (b, n) {
 };
 
 module.exports.shouldSkipPnpm = function () {
-  const REQUIRED_MAJOR_VERSION = 14;
-  const REQUIRED_MINOR_VERSION = 19;
+  // pnpm 8 requires at least Node.js v16.14
+  const REQUIRED_MAJOR_VERSION = 16;
```

Comment:
Shouldn't we check also pnpm version in this check? older pnpm versions also support older node versions